keymanager/src/churp/handler.rs

```diff
@@ -244,18 +245,12 @@ impl Churp {
                 // will detect the polynomial change because the checksum
                 // of the verification matrix in the submitted application
                 // will also change.
-                let dx = threshold.saturating_sub(1);
-                let dy = 2 * dx;
-
-                let dealer = match active_handoff {
-                    0 => Dealer::random(dx, dy, &mut OsRng),
```

Comment:
Before, I wasn't sure if I should increase the round number if round fails. But I needed to, so that nodes know that a round failed and they need to apply again. And because of that, this was broken. So the dealing round (the first one) is always the one, that has empty committee from the previous rounds (handoffs).